HarfangLab/harfanglab/ingest/parser.yml

```
@@ -188,13 +188,78 @@ stages:
           event.dataset: "{{json_event.message.log_type}}"
           user.roles: "{{ json_event.message.groups | map(attribute='name') | join(',') | replace('\n', '') | replace(' ', '') }}"
           organization.id: "{{json_event.message.tenant}}"
-          url.original: "{{json_event.message.details_url_request.url}}"
+          url.original: "{{json_event.message.get('source_url') or json_event.message.details_url_request.url}}"
+          process.executable: "{{json_event.message.image_name or json_event.message.source_image or json_event.message.process_image_path}}"
+          process.pid: "{{json_event.message.pid}}"
+          process.command_line: "{{json_event.message.process_commandline}}"
+          action.properties.StackTrace: "{{json_event.message.stacktrace}}"
+          action.properties.EventId: "{{json_event.message.event_id}}"
```

Is `event.code` not more suitable? In the tests, it seems to be the Windows Event ID. In others formats, we put it in `event.code`.
```suggestion
          event.code: "{{json_event.message.event_id}}"
```